Scripts/Spells/Skill Masteries/BardSpells/Tribulation.cs

```diff
@@ -140,8 +140,9 @@ public override void DoDamage(Mobile victim, int damageTaken)
 				
 				double reduce = DamageModifier(victim);
 				
-				damage -= (int)((damage * reduce) * m_SlayerBonus);
```

Comment:
Errors:
 + Spells/Skill Masteries/BardSpells/Tribulation.cs:
    CS0266: Line 144: Cannot implicitly convert type 'double' to 'int'. An expli
cit conversion exists (are you missing a cast?)